src/mono/browser/runtime/runtime.c

```diff
@@ -188,6 +188,32 @@ init_icall_table (void)
 #endif
 }
 
+static void fixupSymbol(char *key, char *fixedName) {
```

Comment:
We have 3 copies of this exact code 
- https://github.com/dotnet/runtime/blob/main/src/tasks/AotCompilerTask/MonoAOTCompiler.cs#L1235
- https://github.com/dotnet/runtime/blob/main/src/tasks/WasmAppBuilder/ManagedToNativeGenerator.cs#L106

Please add a comment to each place referencing others. If you would find a wat to reuse the same for those two C# places, it would be great. Maybe it could something like a separate file referenced in both csproj's.